test/unit/detail/format_html_test.cpp

```diff
@@ -15,8 +15,8 @@ TEST(html_format, empty_information)
     std::string expected;
 
     // Empty html help page.
-    char const * argv0[] = {"./help_add_test --version-check false", "--export-help", "html"};
-    sharg::parser parser0{"empty_options", 3, argv0};
+    std::array argv0 = {"./help_add_test", "--version-check", "false", "--export-help", "html"};
```

Comment:
```suggestion
    std::array argv0{"./help_add_test", "--version-check", "false", "--export-help", "html"};
```

or maybe even `const`?
Also for others

Reply:
Yes, good idea!